### DISCUSSION TITLE
Thanks Etienne

### DISCUSSION THREAD

**Discussion by MaxHalford** [online-ml/river#435] [Show and tell]
Just wanted to use this discussion board to thank @etiennekintzler for adding the first model selection methods based on bandit logic (they're part of the `expert` module). He spent a lot of time on this, was patient, thoughtful, and considerate. Great work 👌

**Comment by smastelini** [comment]
Wonderful work @etiennekintzler! Besides having broad applications, your implementation brings novelty aspects to the table, regarding research!

**Comment by etiennekintzler** [comment]
@MaxHalford  thanks a lot of for these kind words :pray: , it means a lot to me ! 

Thank you also for helping me in developing the ideas behind the implementation and for your review/assistance on the PR as well (you too @smastelini).

> Wonderful work @etiennekintzler! Besides having broad applications, your implementation brings novelty aspects to the table, regarding research!

Thanks a lot ! I am looking forward to having feedbacks from users and/or inputs from researchers to see what can be improved !
